src/App.js

```diff
@@ -3,9 +3,10 @@ import HomePage from "./pages/HomePage";
 import { Routes, Route } from "react-router-dom";
 import ProductListingPage from "./pages/ProductListingPage";
 import Navbar from "./components/NavBar/Navbar";
-import Footer from "./components/Footer/Footer";
 import Signup from "./components/Auth/Signup";
 import Mockman from "mockman-js";
+import Login from "./components/Auth/Login";
```

Comment:
As all of these are coming from the components folder so you could create an `index.js` file in your components folder and export all components from that file. This helps in importing all components together using named imports.

Reply:
yes I will do these refactor in separate pr